modules/core/core-api/src/main/java/com/enonic/xp/repository/CreateRepositoryParams.java

```diff
@@ -13,6 +15,10 @@
 
     private final RepositorySettings repositorySettings;
 
+    private final PropertyTree data;
+
+    private final AttachedBinaries attachedBinaries;
```

Comment:
It might be a problem here.
BinaryAttachments is what users can post (reference + binary)
AttachedBinaries is what users can get (reference + blobKey)

So this API is suitable only for corner case when binary is already attached and we want to reattach it.